### ISSUE TITLE
ON Mac won't run

### ISSUE THREAD

**Issue by carpediemzak** [ahmadudin/electron-instaPy-GUI#8]
the GUI runs ... i can type in username and password ... but cannot select or type in the instaPY path 
![image](https://user-images.githubusercontent.com/32850793/31630071-7f58e758-b283-11e7-98ba-2f36f2f84179.png)

**Comment by carpediemzak** [comment]
can't  even close it

had to force quit

**Comment by timgrossmann** [comment]
@carpediemzak I tested it with Mac as well, runs like charm on my device

**Comment by carpediemzak** [comment]
Do I need to install electron too ?

**Comment by carpediemzak** [comment]
Do I need electron installed ?


Sent from Yahoo Mail for iPhone


On Monday, October 16, 2017, 4:48 PM, Tim Großmann <notifications@github.com> wrote:


@carpediemzak I tested it with Mac as well, runs like charm on my device

**Comment by timgrossmann** [comment]
No

**Comment by carpediemzak** [comment]
weird ... wasn't working yesterday ... today opened the program and all works :)

**Comment by hestuppfodarn** [comment]
I have the same problem as you have.. 
I cannot select the path..

**Comment by carpediemzak** [comment]
close program - sleep - wake up - open program

repeat above a number of times and eventually starts working (you can select the directory/path)

why this happens ... no idea :o ... but starts working after a few cycles

**Comment by calebmanley** [comment]
I am also getting this issue... Very strange.

**Comment by ahmadudin** [comment]
Sorry, i'll leave the debugging to you guys, there is no mac here.

**Comment by pavle93** [comment]
Same problems happens on Windows as well o.O

**Comment by JacobSpruce** [comment]
[On MAC] This happened to me first time. Just force-close program, then open it. Path should be working now!

**Comment by xavierhb** [comment]
@carpediemzak I got that error when running the app from the setup mounted folder. If you move the app to your Applications folder it will work (at least it worked for me), although there's some issue with Chromedriver.

**Comment by brianroach** [comment]
I found the Default Folder X was causing the open dialog to not be shown when trying to set the PATH. Quitting DFX fixed the issue immediately. Give it a try!

**Comment by ryancurrie** [comment]
I am experiencing the same issue.  Cannot select the path, must force quit the program.  I am using OSX 10.13.2.

**Comment by ahmadudin** [comment]
@ryancurrie have you tried mentioned soultions above?

**Comment by Clowning** [comment]
Same issue for me..

**Comment by krubera** [comment]
Experienced this issue with the DMG version (0.5.0). Zip package version works fine.

**Comment by eorione** [comment]
Just place the app inside tje application folder and thats it.

**Comment by ChungminPark** [comment]
yeah just close this program forcely(option+command+esc). And open it again, it worked for me.
